### PR TITLE
Pagination: DS4 retrofit

### LINKED ISSUE #230: Pagination: retrofit responsive updates to DS4
The DS6 pagination had some nice updates that make it's layout much more responsive. Let's retrofit these changes into DS4. I believe the markup remained exactly the same.

### PR DESCRIPTION
**Pull request by seangates**
## Description
- normalized CSS between DS6 and DS4
- docs are normalized to existing examples
- fixed some issues with padding, margins and general layout of the `.pagination__items`

## Context
The retrofit follows the other retrofit work in the other components.

## References
Fixes #230 

## Screenshots
image

image

### CODE REVIEW

File: src/less/pagination/ds6/pagination.less [hunk 1]
```diff
@@ -1,4 +1,129 @@
 @import "../../less/ds6/variables.less";
 @import "../../less/ds6/mixins.less";
 
-@import "pagination-base.less";
+@import "../base/pagination.less";
+
+.pagination a {
```
Comment (sharma46bhawana) [inline src/less/pagination/ds6/pagination.less]: Instead of having color on the link, should we have it on the class?

Comment (seangates) [reply src/less/pagination/ds6/pagination.less]: This is a carry-over from somewhere. Let me look into why I added it.

**Comment by ianmcburnie** [comment]
I think before and after screenshots are useful in PRs like this.

**Comment by seangates** [comment]
@ianmcburnie Sure. I can provide them.

**Comment by sendlo** [comment]
The old design has a hover color on the navigation arrows:
`
.pagination__next:not([aria-disabled="true"]):focus,
.pagination__previous:not([aria-disabled="true"]):focus,
.pagination__next:not([aria-disabled="true"]):hover,
.pagination__previous:not([aria-disabled="true"]):hover {
  color: rgba(153, 153, 153, 0.9);
}
`

**Comment by sendlo** [comment]
The fluid pattern isn't stretching to fill all available space.

**Comment by seangates** [comment]
@sendlo I would need to create a new background SVG with just that color. Let me look into it. DS6 has one.

**Comment by seangates** [comment]
**Before:**
image

**After:**
image

**Comment by seangates** [comment]
@sendlo @sharma46bhawana I've made updates per PR comments. Can you both please take a look?

**Comment by seangates** [comment]
@sendlo Generally, these have been determined by the design system. Is there one in particular to which you're referring?

**Comment by sendlo** [comment]
@seangates 
https://github.com/eBay/skin/blob/master/src/less/pagination/ds6/pagination-base.less#L6
https://github.com/eBay/skin/blob/master/src/less/pagination/ds6/pagination-base.less#L139
https://github.com/eBay/skin/blob/master/src/less/pagination/ds6/pagination-base.less#L140 (weight rather than size)

**Comment by seangates** [comment]
That second one, `em` size, should be `px`. We should fix that. The font weight is hard-coded from specs from DSL.

**Pull request merged**